src/inverseRobot.js

```diff
@@ -4,10 +4,20 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
   // write code here
+  const revese = {};
```

Comment:
`reverse` is a verb
It is a good name for a function but not for a variable storing an object